images/login-consent-server/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#
```

Comment:
we need to publish the image https://github.com/trustbloc/edge-sandbox/blob/master/ci/azure-pipelines.yml#L57

Reply:
fixed